File: Backend.Net/BusinessDomain/typescript/RailwayBill.ts


module App { 
    export class RailwayBill { 
        public forwardingNumber: string;
        public forwardingStation: TrainStation;
        public takeOverStation: TrainStation;
        public receivingStation: TrainStation;
        public deliveryStation: TrainStation;
        public takeOverDate: string;
        public issueDate: string;
        public issueLocation: string;
        public descriptions: string[];
        public classOfReservation: string;
        public reservationNumber: string;
        public wagon: Wagon;
    }
} 

File: Backend.Net/BusinessDomain/RailwayBill.cs
using System;
using System.Collections.Generic;

namespace BusinessDomain
{
    public sealed class RailwayBill : IDomainObject
    {
        public int ForwaringNumber { get; set; }

        // public int Evu { get; set; }

        public TrainStation ForwardingStation { get; set; }

        public TrainStation TakeOverStation { get; set; }

        public DateTime TakeOverDate { get; set; }

        public TrainStation ReceivingStation { get; set; }

        public DateTime IssueDate { get; set; }

        public string IssueLocation { get; set; }

        public IEnumerable<string> Descriptions { get; set; }

        public int ClassOfReservation { get; set; }

        public string ReservationNumber { get; set; }

        public Wagon Wagon { get; set; }
    }
}